mangle.json

```diff
@@ -56,6 +56,9 @@
       "$_subscribe": "S",
       "$_unsubscribe": "U",
       "preact": "",
+      "$_signal": "s",
+      "$_updater": "_$u",
```

Comment:
I think these need to be double underscores so they don't get reminified?

Reply:
Oh, right, sorry about that 🙂 Opened #176 to fix it.